### PR TITLE
ADD: create speed CDF request handler

### PR DESCRIPTION
**Pull request by youralien**
Test:  the speed cdf png file gets saved in {project_dir}/final_images folder

### CODE REVIEW

File: app/handlers/createSpeedCDF.py [hunk 1]
```diff
@@ -1,22 +1,83 @@
...
+                    'traceback': lines,
+                }
+            }))
+        else:
+            self.finish(json.dumps({
+                'error': {
+                    'code': status_code,
+                    'message': self._reason,
+                }
+}))
```
Comment (runnersaw) [inline app/handlers/createSpeedCDF.py]: This needs indentation?

File: app/handlers/createSpeedCDF.py [hunk 2]
```diff
@@ -1,22 +1,83 @@
...
+import os
 import tornado.web
+from trafficcloud.app_config import get_project_path, get_project_video_path
+from trafficcloud.video import get_framerate
+from trafficcloud.plotting.visualization import vel_cdf
 
-class CreateSpeedCDFHandler(tornado.web.RequestHandler):
+import json
+import traceback
+class MyAppBaseHandler(tornado.web.RequestHandler):
```
Comment (runnersaw) [inline app/handlers/createSpeedCDF.py]: This should probably be in its own file right?

**Review by runnersaw — APPROVED** [review]
🚢 it

File: app/handlers/createSpeedCDF.py (hunk 2 above)
Comment (jabb1123) [reply app/handlers/createSpeedCDF.py]: Agreed

**Review by jabb1123 — DISMISSED** [review]

**Pull request merged**